build.sh

```diff
@@ -159,6 +162,7 @@ USE_STAROS=OFF
 BUILD_JAVA_EXT=ON
 OUTPUT_COMPILE_TIME=OFF
 WITH_TENANN=ON
+WITH_RELATIVE_PATH=OFF
```

Comment:
make it `ON` by default in `build.sh`

change the `run-be-ut.sh`, make it `OFF` in that script.

Reply:
Fixed.